tensorboard/webapp/metrics/effects/metrics_effects_test.ts

```diff
@@ -21,7 +21,7 @@ import {getActivePlugin} from '../../core/store';
 import * as coreTesting from '../../core/testing';
 import {DataLoadState} from '../../types/data';
 import {TBHttpClientTestingModule} from '../../webapp_data_source/tb_http_client_testing';
-import {of, Subject} from 'rxjs';
+import {EMPTY, of, Subject} from 'rxjs';
```

Comment:
There appears to be some significant refactoring in this CL mixed in with the new tests. Can we split the preliminary refactoring to a different PR?

Reply:
Would splitting them into two separate commits help you review the PR? Also, did you turn on the "Hide whitespace changes" option on the GitHub? It is not a "refactor" and more of an indentation change :\

Reply:
Thanks for the tip. Indeed it was just whitespace. I totally missed your earlier comment about it, too.